tests/integ/modin/test_unimplemented.py

```diff
@@ -117,7 +117,7 @@ def test_unsupported_dataframe_binary_methods(func, func_name, caplog) -> None:
     "func, func_name",
     UNSUPPORTED_BINARY_METHODS,
 )
-@sql_count_checker(query_count=1)
+@sql_count_checker(query_count=0)
```

Comment:
Before this PR, we would execute a query to add the entire `other` series to the error message. After this PR, we don't include the other series in the error message.